src/engines/html5/media-source/adapters/native-adapter.js

```diff
@@ -305,7 +305,12 @@ export default class NativeAdapter extends BaseMediaSourceAdapter {
         this._eventManager.listen(this._videoElement, Html5EventType.ABORT, () => this._clearHeartbeatTimeout());
         this._eventManager.listen(this._videoElement, Html5EventType.SEEKED, () => this._syncCurrentTime());
         // Sometimes when playing live in safari and switching between tabs the currentTime goes back with no seek events
-        this._eventManager.listen(window, 'focus', () => setTimeout(() => this._syncCurrentTime(), BACK_TO_FOCUS_TIMEOUT));
+        this._eventManager.listen(window, 'focus', () =>
+          setTimeout(() => {
+            this._videoElement.currentTime = this._videoElement.currentTime > 0.1 ? this._videoElement.currentTime - 0.1 : 0;
```

Comment:
one last thing is, we try to avoid fudge numbers inline, please move to const in top of file(see convention) 
`BACK_TO_FOCUS_NUDGE`

Reply:
Maybe also add a one liner comment just above the current time assignment to explain this bizarre behaviour we are doing - something like - "In IOS when coming back from lock screen hls sometimes gets stuck until seeking" ?